pkgs/development/web/bun/default.nix

```diff
@@ -92,7 +92,7 @@ stdenvNoCC.mkDerivation rec {
       lgpl21Only # javascriptcore and webkit
     ];
     mainProgram = "bun";
-    maintainers = with maintainers; [ DAlperin jk thilobillerbeck cdmistman coffeeispower ];
+    maintainers = with maintainers; [ DAlperin jk thilobillerbeck cdmistman coffeeispower diogomdp ];
```

Comment:
Whenever adding yourself as maintainer, respect alphabetical order. I understand not everyone does that. But it's good if you do.